src/platforms/apple/common/configuration/metric-kit.mdx

```diff
@@ -15,8 +13,9 @@ The MetricKit integration subscribes to:
 - [MXCPUExceptionDiagnostic](https://developer.apple.com/documentation/metrickit/mxcpuexceptiondiagnostic)
 
 The SDK supports this feature from iOS 15 and up and macOS 12 and up because in these versions, MetricKit delivers diagnostic reports immediately, which allows the Sentry SDK to apply current data from the scope.
+Note that the Cocoa SDK has no control over the stacktraces provided by MetricKit. Some stacktraces from MetricKit only offer a couple of frames. If these are not useful to you, you can drop such events in <PlatformLink to="/configuration/filtering/#filtering-error-events/">`beforeSend`</PlatformLink>. You can identify MetricKit events by looking at the `mechanism.type` of the `exception`, which are `mx_hang_diagnostic`, `mx_cpu_exception`, and `mx_disk_write_exception`.
```

Comment:
```suggestion
The SDK supports this feature from iOS 15 and up and macOS 12 and up because in these versions, MetricKit delivers diagnostic reports immediately, which allows the Sentry SDK to apply current data from the scope.

Note that the Cocoa SDK has no control over the stacktraces provided by MetricKit. Some only offer a couple of frames. If these aren't useful to you, you can drop them in <PlatformLink to="/configuration/filtering/#filtering-error-events/">`beforeSend`</PlatformLink>. 

You can identify MetricKit events by looking at the `mx_hang_diagnostic`, `mx_cpu_exception`, and `mx_disk_write_exception` `mechanism.type`.
```